collectors/APICalls/breakpoints.js

```diff
@@ -11,6 +11,9 @@ const breakpoints = [
             {name: 'sessionStorage'},
             {name: 'indexedDB'},
             {name: 'name'}, // can pottentially be used to link two tabs
+            {name: 'navigator.xr'},
+            {name: 'navigator'},
```

Comment:
I wonder why monitoring the whole navigator object access is useful? We already monitor specific methods and props and we shouldn't mark access to apis like `navigator.serviceWorker` or `navigator.sendBeacon` as fingerprinting IMO.